### DISCUSSION TITLE
RFP: Improvements for FIL-RetroPGF-2

### DISCUSSION THREAD

**Discussion by SHSR2001** [filecoin-project/community#711] [RetroPGF]
## Introduction

FIL-RetroPGF-1 was the first in an evolving series of experiments to establish retroactive impact-based funding of public goods projects in the Filecoin ecosystem. For the first round, **224,000 FIL** was pledged by four round funders, **106** eligible applications were received and attested to the Filecoin chain, **25** badgeholders cast their votes, and **99** projects finally met quorum qualifying to receive funding. 

After the round, quantitative and qualitative feedback was collected from applicants, badgeholders, funders, and round operators - a summary of which is available for preview in the report [FIL-RetroPGF-1 Review & Improvements for FIL-RetroPGF-2](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Review-Improvements-for-FIL-RetroPGF-2-c9664fa5b6954629a0d0579b81f7d6a5). Furthermore, the report sets out improvements to consider for Round 2 based on what has been learnt so far. 

As, we progress to the next round, the FIL-RetroPGF Team intends to provide a mechanism for collecting collaborative community input on what should be changed in the upcoming round, and allow for a more decentralized RetroPGF by strengthening the influence the broader community has on the round operations. To enable this, we are introducing an experimental "Request for Proposals" which is a more formal process to hear ideas from the community, on how we can change subsequent RetroPGF rounds (inspired from EIPs, and FIPs). Although, we are excited to introduce this, it is important to understand that approval of proposals ultimately lies solely with the FIL-RetroPGF Ops team, until a more comprehensive governance structure is in place. 

## Themes

Here are a list of themes to inspire you:
- **Principles and Goals**: What changes can be implemented to better align future rounds of FIL-RetroPGF with the broader mission of public goods funding?
- **Software**: What changes can be implemented in the software used by participants in the round, to enhance the user experience for the coming rounds?
- **Operations**: What changes can be implemented in the processes of the round to enhance the user experience for future rounds?
- **Communication**: What changes can be made to improve communication between round operators and participants?
- **Mechanism**: What changes can be made to the voting and funding mechanisms of FIL-RetroPGF to enhance the impact on funding Filecoin's public goods in future rounds?
- **Governance**: What steps can we take to establish a clearer, more robust, and more decentralized governance system for making decisions related to the design and execution of future rounds?

For any proposal to be successful, it needs to be:
- Aligned with [Vision and Mission of FIL-RetroPGF](https://fil-retropgf.notion.site/FIL-RetroPGF-4b6f5358440043c8bb1bf53f0297541e)
- As good as, or better than the current systems in place (explain why and how)

### Submit a Proposal
If you want to propose a change that you'd like to see in the upcoming RetroPGF rounds, open a discussion in the `RetroPGF`  and follow this below format

```
-----Summary-----
__50 word summary of the proposal.__

-----Motivation & Rationale-----
__Below are some useful prompts__
- How would this change allow the initiative to add more value to the Filecoin ecosystem? 
- How is this solution better than the existing setup?
- How is this change in line with the vision and mission of Filecoin RetroPGF?

-----Implementation-----
__Below are some useful prompts__
- What actions from the operators are required to implement the proposal for the next RetroPGF round?
- Which entity will be responsible for effectively implementing the proposal among those involved in running FIL RetroPGF?
- Has this been implemented in any other ecosystem and initiatve?  If so, please describe how it went.

-----Links & Additional Information-----
__Any additional information__
```

### Join the Discussion
Even if you don't have a specific idea to present, we highly encourage everyone to participate in discussions by sharing their thoughts and feedback on the submitted proposals, as well as [the changes set out to be implemented in the report](https://fil-retropgf.notion.site/FIL-RetroPGF-1-Review-Improvements-for-FIL-RetroPGF-2-c9664fa5b6954629a0d0579b81f7d6a5). To start, feel free to comment on this post and any subsequent discussion posts, but do be very mindful of our [code of conduct](https://github.com/filecoin-project/community/blob/4b19c5dd8da828dbd417c3ed3427b29b05d42254/CODE_OF_CONDUCT.md)!!!

Inspired by [EIPs](https://github.com/ethereum/EIPs), [FIPs](https://github.com/filecoin-project/FIPs), [DIPs](https://devcon.org/en/dips/)

**Comment by rekhansejal** [comment]
Hey everyone, I’m Sejal from Gitcoin. We've been collaborating closely with the Filecoin RPGF team to run retro funding rounds. Recently, we’ve started discussing an exciting experiment to run alongside the second FIL RPGF round. Below are the details, and I’d love to invite your feedback on the proposal.

### Prediction Markets Implementation in Filecoin Ecosystem: A Parallel Initiative to FIL-RetroPGF Round 2

### Overview

We're considering launching a prediction market experiment during the Filecoin retro round 2, inspired by the concept of Futarchy - a governance model proposed by economist Robin Hanson. This experiment aims to explore innovative ways of enhancing the Filecoin ecosystem's funding process by leveraging the wisdom of crowds and market mechanisms.

### Objectives
1. Assess ecosystem engagement with prediction markets
2. Provide additional signals for:

- Badgeholders
- External proactive funders
- Projects (on their perceived value)

3. Increase meaningful participation and engagement with FIL ecosystem
4. Gather community insights on the effectiveness of prediction markets in our ecosystem

### Proposed Platform
1. Utilizing [Manifold](https://manifold.markets/home?s=score&f=open&ct=ALL&p=0&fy=1&mt=00000&sw=0), an established prediction market platform
2. Reasons: Manifold operates using a play-money system, which avoids potential legal issues and offers ease of implementation given the desire to test futarchy before creating a custom solution

### Proposed Market Structure
1. Run small curated markets with 3-5 grantees per market
a. Example market: "Which top recipients from the last round will receive the highest funding this round?"
b. We're eager to hear your ideas for other market questions or structures

2. Kick off markets during the week of October 14 and run for 1 week _(subjected to change)_
3. Outcome of the market will be settled based on which grantee in the market receives the most retro funding from RPGF-2
4. Markets will be resolved after rewards are finalized for RPGF-2

### Participation
1. Proposed to be open to all Filecoin ecosystem stakeholders, excluding badgeholders
2. Badgeholders: Observers only, to maintain decision-making integrity
3. We welcome your thoughts on this participation structure

### Tentative Timeline
We will use next week to gather community feedback on this proposal. Our aim is to launch the prediction market experiment immediately following the feedback period.

We're excited about the potential of this experiment, but we need your help to make it as effective and valuable as possible.

**Reply by elizabeth-griffiths** [reply]
I'm currently part of the Funding the Commons residency with @rekhansejal , and we've been discussing and learning a lot about different funding models and how to reduce bias in decision-making. With that in mind, I’m excited to see the introduction of prediction markets tied to the RPGF, which could provide a fresh, engaging way to gather ecosystem-wide support. I believe the prediction markets would be most effective when directly linked to RPGF, ensuring that market outcomes are grounded in real, tangible funding results. This structure would not only increase relevance but also help mitigate biases by incorporating a broader range of stakeholder perspectives in governance and funding decisions.

While I'm supportive of this innovative approach, I want to clarify that it should be introduced in the future as an addition to, not a replacement for, the planned retro funding round 2. This ensures that current processes remain intact while still allowing for further experimentation.

An additional enhancement I’ve recently learned about involves transitioning to a formal impact metric framework. The model described in Relentless Monetisation: The Robin Hood Rules for Smart Giving could be a strong fit. For example, during the prediction market stage, projects could propose hypotheses around impact metrics, and at the RPGF stage, actual metrics would be reported, fostering greater accountability and transparency.

Here’s a real-life example of how such impact metrics come to life: [Nourished Justice - Simaria's Meal Victory](https://app.voicedeck.org/reports/nourished-justice-simarias-meal-victory). One standout feature of this framework is the inclusion of counterfactual analysis, which quantifies the value had the project not delivered the expected outcome, further clarifying the project's true impact.

This dual-stage approach would enrich the insights gained from prediction markets while promoting an outcome-driven culture that benefits all stakeholders in the Filecoin ecosystem.
